### PR TITLE
add test.py

### PR DESCRIPTION
**Pull request by ZhangXinNan**
add test.py . You can input an local image to test zbarlight

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+import argparse
+from PIL import Image
+import zbarlight
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--image', default='./tests/fixtures/two_qr_codes.png', help='input image')
```
Comment (fbochu) [inline test.py]: I think that this argument should be mandatory (remove the leading `--`)

File: test.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
+import argparse
+from PIL import Image
+import zbarlight
+
+def get_args():
```
Comment (fbochu) [inline test.py]: You should merge this into the `main()` function

**Comment by ZhangXinNan** [comment]
@fbochu 
>The second one is to exclude the module from packaging (in MANIFEST.in), this break zbarlight quality tests.

How to do it ?

**Comment by fbochu** [comment]
@ZhangXinNan: You can just append the name of the file to the last line of the MANIFEST.in file -> https://github.com/Polyconseil/zbarlight/blob/master/MANIFEST.in#L7

**Comment by ZhangXinNan** [comment]
@fbochu Please review it , thanks !

File: run.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (fbochu) [inline run.py]: You should add execution bit to the file and a shebang (`#!/usr/bin/python`)

File: run.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import argparse
+from PIL import Image
+import zbarlight
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('image', default='./tests/fixtures/two_qr_codes.png', help='input image')
```
Comment (fbochu) [inline run.py]: The `default` parameter is useless when using positional arguments.

**Pull request merged**